usage.md

```diff
@@ -0,0 +1,106 @@
+Make sure to test the setup will all the desired markers and check if the OptiTrack system can distinguish and localize
+them well. **Let *Motive* run in the background.**
+
+#### TPS Calibration
+
+Once the TPS Transmitter is turned on and the USB Receiver is plugged into the computer directly, run *Chameleon TVR* as
+administrator and let it connect to the sensors. Select the profile (1 or 2) depending on the transmitter that is used (
+they should be labeled). Additionaly, connect the Calibration Base to the USB Hub. Then, in the tool bar, go
+on `Calibration` &rarr; `Calibrate FingerTPS System`. Specify in the GUI where the sensors are placed and begin the
+calibration process by clicking on `begin` and by following the instructions. To ensure that the calibration was
+successful, go to `C:` &rarr; `Program Files` &rarr; `Pressure Profile Systems` &rarr; `Chameleon TVR` &rarr; `setup`
+and check if the modified date of the `-cal` file corresponds to the current time. Close *Chameleon* afterwards.
+
+#### EMG
+
+Disconnect the EMG Probes from power supply such that they start blinking, turn on the EMG Receiver and connect it to
+the USB Hub.
+
+## Recording
+
+**NOTE**: `CTRL+C` might not be working on Windows. Use `CTRL+FN+B` instead. If you find yourself in debug mode in the
+terminal, you can exit that by typing `exit`.
+
+1. Start the *Anaconda Navigator*. Multiple PowerShell Prompts will have to be launched from there.
+
+2. In the first terminal (T1), run
+   ```
+   cd .\path\to\recordings\surgeon_recording\
+   python .\copy_tps_calibration.py
+   ```
+   to copy the calibration file to the correct directory. If this is unsuccessful, the calibration didn't work.
+
+3. Still in T1, run
+   ```
+   cd .\source\emgAcquire\
+   .\bin\emgAcquire.exe config.json
+   ```
+   to start the EMG. This will open a GUI. Click `OK`, wait for connection and check in T1 if data is transmitted. **Do
+   not close T1**.
+
+4. Start your editor, navigate to `path\to\recordings\surgeon_recording\source\config\` and open `sensor_parameters.jon`
+   . Go through the file and adapt the parameters for your tracking setup. In particular, update the list of OptiTrack
+   frames and TPS fingers with a label and the corresponding streaming ID. The streaming ID of the OptiTrack frames can
+   be found in *Motive*, the one for the fingers is assigned during the calibration of the TPS sensors in the UI. Make
+   sure the desired sensors have their status set to `on`.
+
+5. Open the *watchmakingdatalog.sln* project in Visual Studio. In the tool bar, click on `Local Windows Debugger`.
+   **This will open a prompt, do not close it until you finished recording.**
+
+
+6. Open a second terminal (T2), run
+   ```
+   cd path\to\recordings\surgeon_recording\
+   conda activate surgeon_recording
+   python start_all_sensors.py
+   ```
+
+Check the output in T2 to see if the recording is successfully started.
```

Comment:
The EMG periodically prompts the frequency at which it receives data. If you see a recurring message `received at 17Hz` then it works properly. Sometime, the EMG crashes (happen most of the time at startup)  and this need be restarted. You can check whether the periodic message is still printing and if you are seeing any error message. This is a good sign that the EMG is still working.